docs/guides/build-notifications.rst

```diff
@@ -1,13 +1,43 @@
 Enabling Build Notifications
 ============================
 
+Using email
+-----------
+
 Read the Docs allows you to configure emails that can be sent on failing builds.
 This makes sure you know when your builds have failed.
 
-Take these steps to enable build notifications:
+Take these steps to enable build notifications using email:
 
 * Going to **Admin > Notifications** in your project.
 * Fill in the **Email** field under the **New Email Notifications** heading
 * Submit the form
 
-You should now get notified when your builds fail!
+You should now get notified on your email when your builds fail!
+
+Using webhook
+-------------
+
+Read the Docs also allows webhooks configuration to receive notification regarding builds fails.
+
+Take these steps to enable build notifications using webhook:
+
+* Going to **Admin > Notifications** in your project.
+* Fill in the **Url** field under the **New Webhook Notifications** heading
+* Submit the form
+
+The project name, slug and its build instance that failed will be sent as POST request to your webhook url:
```

Comment:
'its build instance' could be slightly clearer as 'details for the build'

also: 'url' -> 'URL'